### DISCUSSION TITLE
Twitter API Kaynaştırması [Resimli Örnek]

### DISCUSSION THREAD

**Discussion by Icaxus** [acikkaynak/deprem-yardim-projesi#38] [Ideas]
Pek çok arkadaşımız dile getiriyordu. Ben de aklımdakileri iletmek istedim. Toparlayarak kısa ve öz anlatmaya çalışacağım. Benzer projeleri hazır olanlar aşağıda belirtebilirler.

1.  **Twit Oluşturma:** Sisteme girilen her bilgi, sistem için oluşturulmuş bir Twitter hesabından, ilgili form verilerinden oluşan -bir resim haline getiren sistemcik ile karakter sınırı aşırtılabilir-  paylaşılacak. _(**Örneğin;** *enkaz formu dolduruldu, *önceden eklenmiş bir enkaz bilgisine cevap yazıldı, *yardım tırının yola çıkma formu, *yardım tırı bilgisinin güncelleme formu, vb.)_
2.  **İçeriğin Kategorisi:** Bu şekilde oluşturulan twit'lerin yazılı kısmı (üstteki bilgilerin medya olarak hazırlandığını varsayıyorum) sistemdeki form türüne göre bir sınıflandırma bilgisi ile başlayacak. _(**Örneğin;** Yeni *yardım-aracı, *enkaz bilgisi, *sıcak-yer-bildirimi, *gıda-destek-talebi, *gıda-destek-bölgesi)_
3. **Mekan Bilgisi:** Bu twit'lerin ilk bilgisi olmalı. Şehrin yeterli olacağını düşünüyorum.
4. **Ek Olarak:** Daha kolay görülebilmesi için _1._ veya _2._ maddedeki alanlara tarih/saat bilgisi de eklenebilir.

![örnek-twit](https://user-images.githubusercontent.com/3382532/218198035-9bd4c21e-ce9b-4549-8b0c-a154b7a4e64e.jpg)

**Comment by sercangoger** [comment]
whatsapp botlar ile de bu istedikileriniz yapılabilir galiba böylesi daha iyi olur gibi sadece fikir.

**Reply by Icaxus** [reply]
Çok teşekkürler. Paylaşım adına daha verimli olabilir whatsapp. ama erişim temel alındığında twitter daha çok kişiye ulaştırır sanıyorum.

**Reply by sercangoger** [reply]
ben teşekkür ederim fikriniz şahane. fakat neredeyse herkes whatsapp kullanıyor ama herkes twitter kullanmıyor gibi. o yüzden ulaşım kapsamı açısından daha iyi olur diye düşündüm sadece.
tekrardan çok teşekkür ederim.
tartışarak fikirlerimizi olgunlaştıralım.
saygılar.

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Ancak Legal ekibi ile değerlendirmemiz sonrasında reddedildi.
"Legal ret, bu konuda lojistik projesi var ve kapalı bir uygulama olarak onay alınması sonrası mümkün aksi takdirde twitter üzerinde paylaşılması suistimale açık"
